server/historian/packages/historian-base/package.json

```diff
@@ -23,7 +23,7 @@
     "tslint": "tslint 'src/**/*.ts'"
   },
   "dependencies": {
-    "@fluidframework/common-utils": "^0.26.0",
+    "@fluidframework/common-utils": "^0.27.0-0",
```

Comment:
i think you probably want to revert the changes to historian since the server versions haven't been updated here